consoleme/celery_tasks/celery_tasks.py

```diff
@@ -573,8 +573,23 @@ def cache_policies_table_details() -> bool:
     if all_s3_errors:
         s3_errors = json.loads(all_s3_errors)
 
+    allowed_tags = config.get("roles.allowed_tags", {})
+    allowed_arns = config.get("roles.allowed_arns", [])
     for arn, role_details_j in all_iam_roles.items():
         role_details = ujson.loads(role_details_j)
+
+        allowed = True
+
+        actual_tags = {k: v for d in role_details.get("Tags") for k, v in d.items()} # Convert List[Dicts] to 1 Dict
+        if allowed_tags and not allowed_tags.items() <= actual_tags.items():
```

Comment:
From my understanding of this line: 
1. If ALL tags present in allowed_tags should also be present in the role's tags (i.e. if allowed_tags has multiply key-value pairs, it is an AND requirement for the allowed tags)
2. The role is allowed to have other tags, other than in the allowed list?

Is that correct?
If so, may want to add a comment before this line 😄

Reply:
If `allowed_tags` is defined, then all tags in `allowed_tags` need to be in `actual_tags`. 
It does not matter if `actual_tags` has other tags as well.

Hehe, i think that answers the questions, let me know if I left you hanging or anything